business/workloads.go

```diff
@@ -526,11 +527,20 @@ func parseLogLine(line string, isProxy bool, engardeParser *parser.Parser) *LogE
 	}
 
 	// override the timestamp with a simpler format
-	timestamp := fmt.Sprintf("%d-%02d-%02d %02d:%02d:%02d",
+	precision := strings.Split(parsedTimestamp.String(), ".")
+	var milliseconds string
+	if len(precision) > 1 {
+		ms := precision[1]
+		milliseconds = ms[:3]
+	} else {
+		milliseconds = "000"
+	}
+
+	timestamp := fmt.Sprintf("%d-%02d-%02d %02d:%02d:%02d.%s",
```

Comment:
For some reason, all my spans doesn't have a timestamp - Tested also with the master branch. Is there anything to set for this? 
![image](https://user-images.githubusercontent.com/49480155/181465686-fa0de0f0-3ac7-49e2-8253-de6ebe5a4298.png)

Reply:
Regarding your comment, you mean that the log lines has different precision?

I think the timestamp is just processed for sorting, but then it print the log line - I think. 
This is the master version: 
![Screenshot from 2022-07-28 08-31-52](https://user-images.githubusercontent.com/49480155/181476314-ac07d204-3c22-4a01-b307-25ed00d3453d.png)

And this branch version: 

![Screenshot from 2022-07-28 08-28-44](https://user-images.githubusercontent.com/49480155/181476416-c13fcd90-6c4a-4834-8293-6672cb44cf1a.png)

Reply:
To see the timestamps you would need to use the "Show Timestamps" option in the kebab menu.

This is a screenshot that has the opposite option because timestamps are enabled, but when it is disabled, the "show" option should replace the "remove" option:

![image](https://user-images.githubusercontent.com/23639005/175380330-3915bef3-fb39-4280-bc59-673a9d91c670.png)

When timestamps are enabled, you should see timestamps for spans, as well as the issue I'm saying about the unpadded milliseconds.

Reply:
Thanks, @israel-hdez ! I will review with this option.

Reply:
Commit change to set correctly the spans precision